iOS_SDK/OneSignalSDK/OneSignalOutcomes/Source/OutcomeEvents/OSSessionManager.h

```diff
@@ -40,6 +40,10 @@
 
 + (void)resetSharedSessionManager;
 
++ (NSTimeInterval)getTimeFocusedElapsed;
+
++ (void)setLastOpenedTime:(NSTimeInterval)lastOpened;
+
```

Comment:
I think these should not be static methods. If we look at calls to session manager, they are always accessed by a shared instance rather than usage as a static class.

Reply:
Done